README.md

```diff
@@ -1,2 +1,15 @@
 # project-1-html-css
-Cloning a New York Times article
+In this project I have practically applied what I have learnt about flexbox, floats and positioning by cloning a New York Times article using a variety of techniques. This clone page is divided into several sections which each have uniques styles mimmicking the appearance of the original article. An effort was also made to use semantic HTML elements and to minimize errors in the code as much as possible.
+
+# Built with:
+- HTML
+- CSS
+
+# Getting Started
+- Download a copy of the project to your local machine.
+- Open the project folder.
+- Open the 'index.html' file in your browser.
+
+# Author:
+Yuvraj Manoo
+- Github: @YuvrajM108
```

Comment:
## Improve the README
- [ ] Do you mind adding a screenshot of the page and a live link as well Please feel free to use this [template ](https://github.com/microverseinc/readme-template)as an inspiration.